requirements/base.txt

```diff
@@ -1,4 +1,6 @@
+awscli==1.20.0
 click==8.0.1
+colorama<=0.4.3
```

Comment:
please, leave some hints on why do we need that and how to check whether the problem is resolved 
even (1) comment in this PR and (2) link to it might work